### ISSUE TITLE
Loading Snippets in *.ipynb

### ISSUE THREAD

**Issue by tklosek0** [slaier/matlab-complete#3]
Hello!
I want to ask, if it is possible to load the snippets in *.ipynb files?
Im running an [Matlab_Kernel](https://github.com/Calysto/matlab_kernel) in Jupyter Notebook, and it would be nice to edit the extension to load in *.ipynb files instead of only in *.m files.

I know that it would load if I would use python in *.ipynb, but im currently only working with Matlab.
Optimal would be an entry which I can edit if I want to use Python/Matlab.
I dont know how to edit the extension-file to let the snippets load in that *.ipynb file-extension.

Thanks!

**Comment by slaier** [comment]
It seems that  extension is not working on jupyter notebook.

https://github.com/microsoft/vscode/issues/34739

**Comment by tklosek0** [comment]
No, Im already running an Matlab_Kernel inside an Jupyter Notebook in Vscode.
My question was, if it is possible to load your extension also with *.ipynb file-types.

For *.m files, the extension offer all snippets, but I want them also in *.ipynb files.
How do I activate the Snippets-Extension on other-filetypes too?

**Comment by tklosek0** [comment]
I think it might be necessary to edit following file [Package.json](https://github.com/Slaier/MatlabSnippets/blob/master/package.json)
in Line 21,27.

**Comment by slaier** [comment]
Please download release [MatlabSnippetsPY-x.x.x.vsix](https://github.com/Slaier/MatlabSnippets/releases/latest)
[](url)
and follow this answer: https://stackoverflow.com/a/62061334

> It is not possible to use user snippets in the Jupyter extension (yet), you can however use interactive python. When you write # %% in a normal .py file this is like opening a new Jupyter cell. You can convert a Jupyter notebook into a python file and it will use interactive python. There you will be able to use your snippets.
